packages/vuetify/src/components/VList/VList.tsx

```diff
@@ -223,6 +246,50 @@ export const VList = genericComponent<new <
       ) focus()
     }
 
+    function getNavigationDirection (key: string): 'next' | 'prev' | 'first' | 'last' | null {
+      switch (key) {
+        case 'ArrowDown': return 'next'
+        case 'ArrowUp': return 'prev'
+        case 'Home': return 'first'
+        case 'End': return 'last'
+        default: return null
+      }
+    }
+
+    function getNextIndex (direction: 'next' | 'prev' | 'first' | 'last'): number {
+      const itemCount = items.value.length
```

Comment:
This should work with slot items too, or the documentation needs to state that it doesn't.